llumnix/arg_utils.py

```diff
@@ -89,8 +89,16 @@ def from_cli_args(cls, args: argparse.Namespace) -> 'EngineManagerArgs':
         attrs = [attr.name for attr in dataclasses.fields(cls)]
         # Set the attributes from the parsed arguments.
         engine_manager_args = cls(**{attr: getattr(args, attr) for attr in attrs})
+        cls._check_args(engine_manager_args)
         return engine_manager_args
 
+    @classmethod
+    def _check_args(cls, args):
+        assert args.migration_backend == 'gloo' \
```

Comment:
Add _check_args in EngineManagerArgs means that we can check it inexplicitly no matter what inference backend is(Note that Llumnix is targeted for multiple inference backends in the future). While for engine arguments, because AsyncEngineArgs is defined inside vLLM, so we define _check_args outside the AsyncEngineArgs and call it explicitly in the entrypoints.